REUSE.toml

```diff
@@ -8,12 +8,6 @@ precedence = "aggregate"
 SPDX-FileCopyrightText = "none"
 SPDX-License-Identifier = "CC0-1.0"
 
-[[annotations]]
-path = "website/data/license.json"
```

Comment:
Why no more test data?

Reply:
I see. I'll replace it with the current version instead then (as the website now needs the extra fields in there).